### PR TITLE
Added CmapTools

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type. https://cmap.ihmc.us/docs/license-cmaptools-client.php
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://twitter.com/MattMailaender/status/1671171059301908483
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: us.ihmc.cmap.CmapTools.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        sha256: a614761d89c6b32dd376315ef578fcfce37df58881d1a655d267dfd3223cabc7
+      - type: file
+        path: cmaptools.png
+        sha256: 8bbbdd04f40e94635da0c03d2b9701d9e07834dedd2a42d4782ffa6212cb7986
+      - type: file
+        path: cmaptools.desktop
+        sha256: 1e474da0e36abf62675d3ca93183b289d93d09b8dce81b706f339ab47ba411c8
+      - type: file
+        path: us.ihmc.cmap.CmapTools.appdata.xml
+        sha256: 14b9cb6901d008634ca85ae8a910980d47b71e16cd295afe7e1b2ce3e7b4c10a
```
Comment (hfiguiere) [inline us.ihmc.cmap.CmapTools.yml]:
```suggestion
```

**Comment by Mailaender** [comment]
bot, build us.ihmc.cmap.CmapTools

File: us.ihmc.cmap.CmapTools.yml (hunk 1 above)
Comment (Mailaender) [reply us.ihmc.cmap.CmapTools.yml]: Last-minute changes to OARS.

**Comment by hfiguiere** [comment]
for `type` `file`, you should remove the `sha256`.

File: cmaptools.desktop
Comment (hfiguiere) [inline cmaptools.desktop:1]: this file should be properly name from the get go.

File: cmaptools.png
Comment (hfiguiere) [inline cmaptools.png:1]: this file should be properly name from the get go.

File: us.ihmc.cmap.CmapTools.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.0-->
+  <id>us.ihmc.cmap.CmapTools</id>
+  <name>CmapTools</name>
+  <summary>Allows users to to create and share concept maps.</summary>
+  <developer_name>Florida Institute for Human &amp; Machine Cognition</developer_name>
+  <launchable type="desktop-id">us.ihmc.cmap.CmapTools.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
```
Comment (hfiguiere) [inline us.ihmc.cmap.CmapTools.appdata.xml:9]: no project license ?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: us.ihmc.cmap.CmapTools.yml (hunk 1 above)
Comment (hfiguiere) [reply us.ihmc.cmap.CmapTools.yml]: just remove them.

File: us.ihmc.cmap.CmapTools.appdata.xml (hunk 2 above)
Comment (Mailaender) [reply us.ihmc.cmap.CmapTools.appdata.xml:9]: proprietary freeware

Comment (hfiguiere) [reply us.ihmc.cmap.CmapTools.appdata.xml:9]: it's still missing.

**Comment by Mailaender** [comment]
It is a bit unfortunate that the installer relies on its own JVM.

File: us.ihmc.cmap.CmapTools.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      - rm "${FLATPAK_DEST}/lib/cmaptools/Uninstall CmapTools"
+      - rm ${FLATPAK_DEST}/lib/cmaptools/CmapTools
+      - rm -r ${FLATPAK_DEST}/lib/cmaptools/UninstallerData
+      - install -Dm755 cmaptools.sh ${FLATPAK_DEST}/bin/cmaptools
+      - install -Dm644 us.ihmc.cmap.CmapTools.desktop ${FLATPAK_DEST}/share/applications/us.ihmc.cmap.CmapTools.desktop
+      - install -Dm644 us.ihmc.cmap.CmapTools.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/us.ihmc.cmap.CmapTools.png
+      - install -Dm644 us.ihmc.cmap.CmapTools.appdata.xml /app/share/appdata/us.ihmc.cmap.CmapTools.appdata.xml
+    sources:
+      - type: file
+        url: http://cmapdownload.ihmc.us/installs/CmapTools/Linux/Linux64CmapTools_v6.04_09-24-19.bin
```
Comment (barthalion) [inline us.ihmc.cmap.CmapTools.yml:36]: Is it redistributable?

Comment (Mailaender) [reply us.ihmc.cmap.CmapTools.yml:36]: They sum up their license as free for commercial use.

File: us.ihmc.cmap.CmapTools.appdata.xml (hunk 2 above)
Comment (Mailaender) [reply us.ihmc.cmap.CmapTools.appdata.xml:10]: Sorry, the graphically editor swallowed it.

File: us.ihmc.cmap.CmapTools.yml (hunk 3 above)
Comment (Mailaender) [reply us.ihmc.cmap.CmapTools.yml:36]: The license does not say.

Comment (barthalion) [reply us.ihmc.cmap.CmapTools.yml:36]: Then it should be assumed it's not, and reworked to use extra-data.

Comment (Mailaender) [reply us.ihmc.cmap.CmapTools.yml:36]: Tried that. It seems I don't have Java during the extra-data step. :(

**Comment by hfiguiere** [comment]
bot, build us.ihmc.cmap.CmapTools

File: us.ihmc.cmap.CmapTools.yml (hunk 3 above)
Comment (barthalion) [reply us.ihmc.cmap.CmapTools.yml:36]: You can always try contacting upstream for clarification.

Comment (Mailaender) [reply us.ihmc.cmap.CmapTools.yml:36]: The [FAQ](https://cmap.ihmc.us/docs/cmaptools-faqs) states:

> **Can I get Access to the Source Code of CmapTools?**
No, we do not distribute the source code of CmapTools. The software is not Open Source, although CmapTools is _free to distribute_. Only the executable programs are available at the [CmapTools download](https://cmap.ihmc.us/cmaptools/cmaptools-download) page.

**Comment by barthalion** [comment]
bot, build us.ihmc.cmap.CmapTools

**Comment by barthalion** [comment]
Sorry for the noise, trying to sort out recent merge failures.

**Pull request closed without merging**